picard.wdl

```diff
@@ -113,26 +113,26 @@ task CollectMultipleMetrics {
     }
 
     output {
-        File alignmentSummary = basename + ".alignment_summary_metrics"
-        File baitBiasDetail = basename + ".bait_bias_detail_metrics"
-        File baitBiasSummary = basename + ".bait_bias_summary_metrics"
-        File baseDistributionByCycle = basename + ".base_distribution_by_cycle_metrics"
-        File baseDistributionByCyclePdf = basename + ".base_distribution_by_cycle.pdf"
-        File errorSummary = basename + ".error_summary_metrics"
-        File gcBiasDetail = basename + ".gc_bias.detail_metrics"
-        File gcBiasPdf = basename + ".gc_bias.pdf"
-        File gcBiasSummary = basename + ".gc_bias.summary_metrics"
+        File? alignmentSummary = basename + ".alignment_summary_metrics"
+        File? baitBiasDetail = basename + ".bait_bias_detail_metrics"
+        File? baitBiasSummary = basename + ".bait_bias_summary_metrics"
+        File? baseDistributionByCycle = basename + ".base_distribution_by_cycle_metrics"
+        File? baseDistributionByCyclePdf = basename + ".base_distribution_by_cycle.pdf"
+        File? errorSummary = basename + ".error_summary_metrics"
+        File? gcBiasDetail = basename + ".gc_bias.detail_metrics"
+        File? gcBiasPdf = basename + ".gc_bias.pdf"
+        File? gcBiasSummary = basename + ".gc_bias.summary_metrics"
         File? insertSizeHistogramPdf = basename + ".insert_size_histogram.pdf"
         File? insertSize = basename + ".insert_size_metrics"
-        File preAdapterDetail = basename + ".pre_adapter_detail_metrics"
-        File preAdapterSummary = basename + ".pre_adapter_summary_metrics"
-        File qualityByCycle = basename + ".quality_by_cycle_metrics"
-        File qualityByCyclePdf = basename + ".quality_by_cycle.pdf"
-        File qualityDistribution = basename + ".quality_distribution_metrics"
-        File qualityDistributionPdf = basename + ".quality_distribution.pdf"
-        File qualityYield = basename + ".quality_yield_metrics"
+        File? preAdapterDetail = basename + ".pre_adapter_detail_metrics"
+        File? preAdapterSummary = basename + ".pre_adapter_summary_metrics"
+        File? qualityByCycle = basename + ".quality_by_cycle_metrics"
+        File? qualityByCyclePdf = basename + ".quality_by_cycle.pdf"
+        File? qualityDistribution = basename + ".quality_distribution_metrics"
+        File? qualityDistributionPdf = basename + ".quality_distribution.pdf"
+        File? qualityYield = basename + ".quality_yield_metrics"
         # Using a glob is easier. But will lead to very ugly output directories.
-        Array[File] allStats = select_all([
+        Array[File]? allStats = select_all([
```

Comment:
Okay, every output except this one 😛 
```suggestion
        Array[File] allStats = select_all([
```

Reply:
Haha, I did exactly as you asked :stuck_out_tongue_winking_eye: